### PR TITLE
formula_desc_cop: empty string is not a valid desc

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
-----

Just a slightly tweaked version of https://github.com/Homebrew/brew/issues/3286#issuecomment-334983011

### CODE REVIEW

**Comment by JCount** [comment]
@DomT4 I restarted the macOS Travis CI build job

**Comment by DomT4** [comment]
Thanks @JCount!

File: Library/Homebrew/test/rubocops/formula_desc_cop_spec.rb [hunk 1]
```diff
@@ -27,6 +27,27 @@ class Foo < Formula
       end
     end
 
+    it "When desc is an empty string" do
```
Comment (reitermarkus) [inline Library/Homebrew/test/rubocops/formula_desc_cop_spec.rb]: When using `it`, this should read like a proper sentence, i.e.

```ruby
it "reports an offense when desc is an empty string" do
```

**Comment by MikeMcQuaid** [comment]
Nice work, thanks again @DomT4!

**Comment by DomT4** [comment]
Thanks for the merge! ❤️

**Pull request merged**